design-editor/src/pane/design-editor.js

```diff
@@ -943,7 +943,7 @@ class Model {
         // }
 
 		this._DOM.body.appendChild(scriptRunner);
-		this.addLibrary('helper.js');
+		this.addLibrary('back-button-support.js');
```

Comment:
In this commit you remove helper.js but I don't see back-button-support.js added.

Reply:
Right, I've added this renamed file